webpack/scenes/Subscriptions/__tests__/SubscriptionsReducer.test.js

```diff
@@ -1,85 +1,168 @@
-import * as types from '../SubscriptionConstants';
+import { testReducerSnapshotWithFixtures } from '../../../move_to_pf/test-utils/testHelpers';
```

Comment:
SubscriptionsReducer test migrated to use the `testReducerSnapshotWithFixtures` helper.